src/sentry/models/project.py

```diff
@@ -98,6 +98,7 @@ class Project(Model):
         'pk',
         'slug',
     ])
+    platform = models.CharField(max_length=64, null=True)
```

Comment:
I guess my question really is why do we need to store this here if we have `ProjectPlatform` already? Can't we derive this by querying like, `ProjectPlatform.objects.filter(project=p).first()` instead?

(I also haven't read a lot so I may be missing a detail here)